src/test/java/jewellery/inventory/integration/AuthenticatedIntegrationTestBase.java

```diff
@@ -44,12 +45,14 @@ abstract class AuthenticatedIntegrationTestBase {
 
   @Autowired private ImageService imageService;
   @Autowired private ImageRepository imageRepository;
+  @Autowired private ProductPriceDiscountRepository productPriceDiscountRepository;
 
   protected HttpHeaders headers;
 
   @BeforeEach
   void setup() {
     deleteAllImages();
+    productPriceDiscountRepository.deleteAll();
```

Comment:
we need it because then we try to delete tables in the ProductRepository but we get an error(delete on table "product" violates foreign key constraint "product_price_discount_product_id_fkey" on table "product_price_discount")